src/routes/safe/components/Transactions/helpers/EditTxParametersForm/index.tsx

```diff
@@ -215,7 +213,7 @@ export const EditTxParametersForm = ({
                         name="ethMaxPrioFee"
                         defaultValue={ethMaxPrioFee}
                         type="number"
-                        placeholder={`${fromWei(DEFAULT_MAX_PRIO_FEE, 'gwei')} (GWEI)`}
+                        placeholder="Max priority fee"
```

Comment:
Can we grab a dynamic value here?

Reply:
@DiogoSoaress chose the `placeholder` to show the default value of the field. We needn't show any value in the placeholder if the field doesn't have a fallback.